docs/site/Loopback-component-authentication.md

```diff
@@ -0,0 +1,603 @@
+---
+lang: en
+title: 'Authentication'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Loopback-component-authentication.html
+---
+
+## Overview
+
+Security is of paramount importance when developing a web or mobile application
+and usually consists of two distinct pieces :
```

Comment:
@nabdelgadir 

> Just some nitpicks but 👍. When you're linking to another docs on the website, if you use {file-name}.md, then, in vscode, it goes to the file instead of the website so that's why I recommend changing those (there are more of them in the file that I didn't comment on).

Umm...I cannot find any use of fileName.md in my document.

Reply:
@nabdelgadir and I spoke offline. I wasn't sure what she was asking in her comment above.

I understand now that Nora was stating that I should not reference `.html` files to loopback.io documentation, but that I should instead try to use their respective `.md` files.

I distinctly remember someone telling me that inside that I `should` reference the loopback.io `.html` files.  I've done it before in : https://raw.githubusercontent.com/strongloop/loopback-next/master/docs/site/submitting_a_pr.md .

Both work...

@jannyHou @raymondfeng @dhmlau @b-admike , is there a preference?

Reply:
_I think_ (at least) if the referencing file is coming from loopback-next, it should be `.md`. 
See https://raw.githubusercontent.com/strongloop/loopback-next/master/docs/site/Concepts.md for examples.

Reply:
I prefer to use the relative link. The external url can be changed due to

- A refactor that changes the host: like the PR that Raymond just merged, the apidocs are moved from `apidocs.strongloop.com` to `loopback.io`

- An update of the permanent link metadata like https://github.com/strongloop/loopback-next/pull/2977/files#diff-f95d62f4892a8c070795ee268ae7ca15R6

Pointing to a relative path saves us the updating effort (some times it's huge).

Reply:
relative link for `.md` file works for me. 👍

Reply:
I will use references the .md files for docs instead of html :)